appveyor.yml

```diff
@@ -30,6 +30,6 @@ build_script:
 
 test_script:
   - ps: . ".\Run Tests.ps1"
-        # cd $env:APPVEYOR_BUILD_FOLDER
-        # cargo run --release --example ci_test > output.log
-        # type output.log
+        cd $env:APPVEYOR_BUILD_FOLDER
+        cargo run --release --example ci_test > output.log
+        type output.log
```

Comment:
Let's keep these three disabled, too. It's only about building for now, and doing that on Travis should suffice.